content/200-concepts/100-components/02-prisma-client/052-advanced-type-safety/index.mdx

```diff
@@ -127,4 +127,109 @@ userPosts.posts
 userPosts.profile
 ```
 
-> See the [model query options](/reference/api-reference/prisma-client-reference#model-query-options) reference for more information about the different types available.
+> See the [model query options](../../../../reference/api-reference/prisma-client-reference#model-query-options) reference for more information about the different types available.
+
+### Generated <inlinecode>UncheckedInput</inlinecode> types
+
+The `UncheckedInput` types are a special set of generated types that allow you to perform some operations that Prisma Client considers "unsafe", like directly writing foreign keys. You can choose either the "safe" `Input` types or the "unsafe" `UncheckedInput` type when doing operations like `create`, `update`, or `upsert`.
```

Comment:
Are there other use cases than foreign keys?

Reply:
That's the only one I know, to be honest. I was quoting previous [discussions](https://prisma-company.slack.com/archives/C016KUHB1R6/p1636563619030800?thread_ts=1636563201.030700&cid=C016KUHB1R6) and [support issues](https://github.com/prisma/prisma/discussions/10121). 

Any suggestions on how to rephrase this?